js/ui/apple2.ts

```diff
@@ -306,7 +306,7 @@ export function doDelete(name: string) {
 }
 
 interface LoadOptions {
-    address?: word;
+    address?: word | undefined;
```

Comment:
I'd leave this as optional, but that it can't be explicitly `undefined`.

Reply:
Requires more spread operators, but yeah.